Makefile

```diff
@@ -38,3 +39,10 @@ clean:
 update-submodules:
 	@echo "+\n++ Updating submodules ...\n+"
 	@mkdir -p client && cd ./client && git submodule init && git submodule update
+
+reinitialize-git:
+	@rm .gitmodules
+	@rm -rf .github
+	@rm -rf .git
+	@rm -rf ./client/.git
```

Comment:
Removing files makes me nervous. It works as-expected, but any ideas of where this could go horribly wrong?

It _is_ incredibly convenient